### ISSUE TITLE
LinkError on Safari 14 / macOS

### ISSUE THREAD

**Issue by Technopathic** [iodide-project/pyodide#1159]
Attempting to run Pyodide on Safari / macOS Big Sur results in the console error below:

<img width="864" alt="Screenshot_2021-01-21_at_10 30 45" src="https://user-images.githubusercontent.com/9061683/105329843-45fae200-5bda-11eb-9712-d4326f6fc1bc.png">

Any hint as to what this is related to do or what causes this would be appreciated. Or if possible how it could be fixed. 
Issue does not appear to be occuring on Windows / Chrome or Firefox.

**Comment by rth** [comment]
Yes, it's a known issue reported in https://github.com/iodide-project/pyodide/issues/721. Closing as a duplicate, please comment there instead.
